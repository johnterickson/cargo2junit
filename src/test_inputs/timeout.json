{ "type": "suite", "event": "started", "test_count": 1 }
{ "type": "test", "event": "started", "name": "tests::long_execution_time" }
{ "type": "test", "event": "timeout", "name": "tests::long_execution_time" }
{ "type": "test", "name": "tests::long_execution_time", "event": "ok" }
{ "type": "suite", "event": "ok", "passed": 1, "failed": 0, "allowed_fail": 0, "ignored": 0, "measured": 0, "filtered_out": 0 }
