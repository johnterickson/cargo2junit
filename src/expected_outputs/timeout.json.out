<?xml version="1.0" encoding="utf-8"?>
<testsuites>
  <testsuite id="0" name="cargo test #0" package="testsuite/cargo test #0" tests="1" errors="0" failures="0" hostname="localhost" timestamp="2020-06-15T20:58:07.146949500+00:00" time="0">
    <testcase name="long_execution_time" classname="tests" time="0" />
  </testsuite>
</testsuites>